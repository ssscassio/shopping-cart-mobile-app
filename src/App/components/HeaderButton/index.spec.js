/** @format */

import React from 'react';
import { shallow } from 'enzyme';

import { HeaderButton } from '.';

describe('<HeaderButton />', () => {
  test('should render properly', () => {
    const wrapper = shallow(<HeaderButton iconName="shopping-cart" onPress={jest.fn()} />);
    expect(wrapper).toMatchSnapshot();
  });
});
