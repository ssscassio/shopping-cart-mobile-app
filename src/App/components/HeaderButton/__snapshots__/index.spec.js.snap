// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HeaderButton /> should render properly 1`] = `
<TouchableOpacity
  activeOpacity={0.8}
  onPress={[Function]}
  style={
    Object {
      "alignItems": "center",
      "flexDirection": "row",
      "justifyContent": "center",
      "padding": 10,
    }
  }
>
  <Icon
    allowFontScaling={false}
    color="#fff"
    name="shopping-cart"
    size={24}
  />
  <Badge
    badgeColor="rgba(249, 200, 34, 1.00)"
    textColor="#333"
  />
</TouchableOpacity>
`;
